src/makeCalculator.js

```diff
@@ -35,7 +35,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, value) {
+      this.result = operation(value);
```

Comment:
I'm not sure but maybe you can omit the assigning and use only 'operation(value)', it will work the same

Reply:
Looks good, thanks!